### DISCUSSION TITLE
Segmentation resampling

### DISCUSSION THREAD

**Discussion by function2-llx** [MIC-DKFZ/nnUNet#1129] [Q&A] [answered]
Hi everyone,

I notice that `Preprocessor3DBetterResampling` class uses third-order spline interpolation for segmentation. Does this name imply that it is better than the first-order one used in `GenericPreprocessor` for segmentation?

Thanks.

**Comment by FabianIsensee** [comment, marked as answer]
The difference is immeasurable and the results of Preprocessor3DBetterResampling have more interpolation artifacts (less visually appealing) so it's not the default

**Reply by function2-llx** [reply]
Thank you, Fabian!
